dotnet/frontend/Controllers/GreetingController.cs

```diff
@@ -77,7 +80,11 @@ private async Task<string> GetMessageAsync(HttpClient client)
             using var span = _tracer.StartActiveSpan("✨ call /message ✨");
             var request = new HttpRequestMessage(HttpMethod.Get, GetMessageEndpoint());
             var response = await client.SendAsync(request);
-            if (!response.IsSuccessStatusCode) return "OH NO!";
+            if (!response.IsSuccessStatusCode)

```

Comment:
Is this formatting something we can put in the editorconfig?

Reply:
I _think_ this is a combo of `csharp_prefer_braces` and `csharp_new_line_before_open_brace = all`, but the former is set to silent, so I can change that.